<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Observatory</class>
 <widget class="QWidget" name="Observatory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Observatory</string>
  </property>
  <property name="toolTip">
   <string>Status of the Observatory</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QGroupBox" name="statusDefinitionBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Observatory Status</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QCheckBox" name="useShutterCB">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If selected, the shutter needs to be open for the observatory status being &amp;quot;READY&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Shutter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="useWeatherCB">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If selected, the weather needs to be OK for the observatory status being &amp;quot;READY&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Weather</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="useDomeCB">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If selected, the dome needs to be unparked for the observatory status being &amp;quot;READY&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Dome</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="statusReadyButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close the shutter of the dome. For advanced control of the dome please use the INDI tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked
{
background-color: maroon;
border: 1px outset;
font-weight:bold;
}</string>
        </property>
        <property name="text">
         <string>READY</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="domeBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Dome</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="domePark">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Park the dome. For advanced control of the dome please use the INDI tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked
{
background-color: maroon;
border: 1px outset;
font-weight:bold;
}</string>
        </property>
        <property name="text">
         <string>PARK</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="domeUnpark">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Unpark the dome. For advanced control of the dome please use the INDI tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked
{
background-color: maroon;
border: 1px outset;
font-weight:bold;
}</string>
        </property>
        <property name="text">
         <string>UNPARKED</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <layout class="QHBoxLayout" name="domeAngleBox">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="angleLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Angle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="domeAngleSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setDomeAngleButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5" alignment="Qt::AlignTop">
    <widget class="QGroupBox" name="weatherBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Weather</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="weatherLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Display the weather status. The warning and alert limits are set in the INDI tab.</string>
        </property>
        <property name="text">
         <string>Weather Status: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="weatherStatusLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" alignment="Qt::AlignTop">
       <widget class="QGroupBox" name="weatherActionsBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignTop">
          <widget class="QGroupBox" name="weatherWarningBox">
           <property name="title">
            <string>War&amp;ning</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="weatherWarningDomeCB">
              <property name="text">
               <string>park dome</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="weatherWarningShutterCB">
              <property name="text">
               <string>close shutter</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="weatherWarningSchedulerCB">
              <property name="text">
               <string>stop scheduler</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="weatherWarningStatusLabel">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Status: inactive&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="weatherWarningDelayLabel">
                <property name="text">
                 <string>delay (sec):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="weatherWarningDelaySB">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QGroupBox" name="weatherAlertBox">
           <property name="title">
            <string>Ale&amp;rt</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QCheckBox" name="weatherAlertShutterCB">
              <property name="text">
               <string>close shutter</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="weatherAlertDomeCB">
              <property name="text">
               <string>park dome</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="weatherAlertSchedulerCB">
              <property name="text">
               <string>stop scheduler</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="weatherAlertStatusLabel">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Status: inactive&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::AutoText</enum>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="weatherAlertDelayLabel">
                <property name="text">
                 <string>delay (sec):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="weatherAlertDelaySB">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="shutterBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Shutter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="shutterClosed">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close the shutter of the dome. For advanced control of the dome please use the INDI tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked
{
background-color: maroon;
border: 1px outset;
font-weight:bold;
}</string>
        </property>
        <property name="text">
         <string>CLOSED</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>16</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="shutterOpen">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>96</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>72</width>
          <height>36</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open the shutter of the dome. For advanced control of the dome please use the INDI tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked
{
background-color: maroon;
border: 1px outset;
font-weight:bold;
}</string>
        </property>
        <property name="text">
         <string>OPEN</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
